src/js/components/DataTable/DataTable.js

```diff
@@ -406,7 +406,9 @@ const DataTable = ({
           )}
         </StyledDataTable>
       </OverflowContainer>
-      {paginate && items && <Pagination alignSelf="end" {...paginationProps} />}
+      {paginate && items && items.length ? (
```

Comment:
I think this might cause a jump in vertical layout if the table has a fixed height but is lazily loaded. I wonder if instead Pagination should treat no items the same a single item, with just one page and disabled controls.

Reply:
Summarizing offline discussion: Pagination controls should only show if the number of items is great than the step (meaning that there would be more than one page).